dashboard/app/views/levels/_dialog.html.haml

```diff
@@ -72,6 +72,19 @@
         %button#ok-button.btn-danger{style: 'float: right;'}
           =t('dialog.startover_ok')
 
+  #instructions-dialogcontent
+    .modal-content.no-modal-icon.markdown-instructions-container
+      %p.dialog-title
+        ="#{t('puzzle')} #{@script_level.try(:position) || 1} #{t('of')} #{@script_level.try(:stage_total) || 1}"
+      .instructions-markdown
+        %p
+          -# CSP is en_us only, always show markdown instructions
```

Comment:
Is this comment accurate? Is `views/levels/_dialog.html.haml` template somehow gated to CSP?